addons/sale_stock/models/account_invoice.py

```diff
@@ -7,10 +7,6 @@
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
 
-    incoterms_id = fields.Many2one('account.incoterms', string="Incoterms",
```

Comment:
Is it explicitly asked to remove that field ?

Reply:
Yes, see task chatter, we have discussed with @qdp-odoo 

and now this is already fixed in master. after rebase, it will remove from diff